test/fixtures.js

```diff
@@ -1,3 +1,5 @@
+const FIXTURE = `fixture`;
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/codingstyle-cn/hlj/pullRequest?prid=7143429)
```suggestion
const FIXTURE = "fixture";
```